pkg/kube/cluster-init.sh

```diff
@@ -641,16 +641,33 @@ get_enc_status() {
     fi
 }
 
+rotate_cluster_token() {
+        local token="$1"
+        /usr/bin/k3s token rotate --new-token "$token"
```

Comment:
Are there any documented error return codes from this path we should handle?

Reply:
I have added some logging, to see the output of k3s command using '$?', just as for debugging purpose.